symfony/phpunit-bridge/3.3/phpunit.xml.dist

```diff
@@ -10,6 +10,7 @@
     <php>
         <ini name="error_reporting" value="-1" />
         <env name="APP_ENV" value="test" />
+        <env name="APP_DEBUG" value="false" />
```

Comment:
We mostly have 4 environments in our application where the additional does match mostly the prod configuration, so I did change this line in the [bootstrap.php](https://github.com/symfony/recipes/blob/d9dc7a8ff5b46d20b1197d14bcf2551a448a8ae1/symfony/framework-bundle/4.2/config/bootstrap.php#L20) to avoid a `in_array` check to  `... && 'dev' === $_SERVER['APP_ENV']`.